sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -274,6 +277,34 @@ private String getAndValidateFullyQualifiedNamespace() {
         return fullyQualifiedNamespace;
     }
 
+    /**
+     * Sets a custom endpoint address when connecting to the Event Hubs service. This can be useful when your network
+     * does not allow connecting to the standard Azure Event Hubs endpoint address, but does allow connecting through
+     * an intermediary. For example: {@literal https://my.custom.endpoint.com:55300}.
+     * <p>
```

Comment:
nit; In JavaDoc, should we use "Service Bus" rather than "Event Hubs"?

Reply:
Sorry wrong type.